css/styles.css

```diff
@@ -216,9 +217,11 @@ button:active {
 #form-section {
   width: 100%;
   display: flex;
```

Comment:
- [ ] Great work on your about me section! However, I noticed that there are some missing spacings between the text and the cards displaying your skills, languages, and frameworks. It would be great if you could add the appropriate spacing in those areas.

- [ ] Additionally, please take a moment to review the elements inside your cards as they may not match the template's intended appearance.

- [ ] Lastly, kindly ensure that each of your cards has the correct spacing within them.

   ### Figma design:
   ![image](https://github.com/flov3rh/portfolio/assets/107952724/6f7b1b18-0ab8-4ed5-91e5-3683ff55d3f8)

   ### Your design:
   ![image](https://github.com/flov3rh/portfolio/assets/107952724/3798f9f6-0121-4b64-a85e-5d54906b522d)

- [ ] Make sure your mobile version looks like before.
   ![image](https://github.com/flov3rh/portfolio/assets/107952724/d9b92f51-7681-44b9-9d61-9b6fbd38f541)